### DISCUSSION TITLE
Software update not recognized while charging

### DISCUSSION THREAD

**Discussion by modalchip** [teslamate-org/teslamate#2101] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

When charging, TeslaMate will not notice an update is occurring. This causes the update installation time to be zero. To have TeslaMate see the update status, charging must be stopped.

### Expected Behavior

Charging and Updating should be recorded simultaneously.

### Relevant log output

```shell
n/a
```

### Type of installation

Docker

### Version

1.24.2

**Comment by adriankumpf** [comment]
Since when does charging continue when an update is installed? According to Tesla's own FAQ charging will be stopped:

> **Will my car charge while installing new software?**
> Your car will not charge while installing an over-the-air update. If you install a software update while charging, your car will stop charging during the update and will automatically resume charging once the update is complete.

https://www.tesla.com/support/software-updates

Furthermore, it would add a lot of complexity. Since updates are also detected by TeslaMate retrospectively, the only benefit would be that the installation time would be recorded instead of being shown as 0.

**Comment by modalchip** [comment]
It's never stopped charging for me, at least while at home. I sometimes like to look back at how long it took to install updates, and stop charging manually to make sure to gather this data. There is a known work around so if it's a difficult change, let's feel free to close the issue.
Thanks!
